tests/components/elgato/test_config_flow.py

```diff
@@ -217,20 +224,22 @@
     flow = config_flow.ElgatoFlowHandler()
     flow.hass = hass
     flow.context = {"source": SOURCE_ZEROCONF}
-    result = await flow.async_step_zeroconf(
-        {"hostname": "example.local.", "port": 9123}
-    )
+    result = await flow.async_step_zeroconf({
+        "hostname": "example.local.",
+        "port": 9123
+    })
 
     assert flow.context[CONF_HOST] == "example.local"
     assert flow.context[CONF_PORT] == 9123
     assert flow.context[CONF_SERIAL_NUMBER] == "CN11A1A00001"
-    assert result["description_placeholders"] == {CONF_SERIAL_NUMBER: "CN11A1A00001"}
+    assert result["description_placeholders"] == {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4635436)